### ISSUE TITLE
Organizer recruiting for hackseq19

### ISSUE THREAD

**Issue by emreerhan** [hackseq/hackseq19#1]
Please comment below or email us at hackseq [at] gmail.com if you are interested in helping organize hackseq19. We are not currently looking for hard commitments.

If you would like to learn more about hackseq, check out previous iterations of this hackathon at www.hackseq.com, and read the [f1000 paper](https://f1000research.com/articles/6-197/v1) about hackseq16.

If you would like to learn more about the responsibilities of being an organizer, check out the [hackseq18 organizational wiki](https://github.com/hackseq/hackseq18/wiki).

**Comment by baraaorabi** [comment]
I'm in!

**Comment by saberhq** [comment]
I will be very happy to help. I'm in as well.

**Comment by nedashokraneh** [comment]
Probably, I'm not here at summer, but I could help remotely if needed. If it is not the problem, I would like to help too.

**Comment by kfgibson** [comment]
I'm also interested in helping!

**Comment by NoushinN** [comment]
I am interested for remote help from Vancouver Island as well if there'e a possibility, perhaps at social media capacity

**Comment by itsDagu** [comment]
Interested in helping! Looking forward to meeting with y'all.

**Comment by oganm** [comment]
Still need people? Also do we also get to code if are organising stuff?

**Comment by emreerhan** [comment]
@oganm we're not looking for any particular number of organizers. Over the last 3 years we've had between 7-10 organizers, and we currently have 8 people who have expressed interest in organizing this year. I believe we can have as many organizers as people willing to contribute, as long as we remain at some reasonable number.

tl;dr the more the merrier!

edit: And yes, you may also take part as either a participant or a team leader.

**Comment by oganm** [comment]
Sounds good to me then.

**Comment by scatcher125** [comment]
I'd love to help organize as well :)

**Comment by alexsweeten** [comment]
Count me in again :)

**Comment by bfjia** [comment]
I'd be interested in organizing again!

**Comment by emreerhan** [comment]
Hello everyone! Thank you for showing interest in organizing hackseq19. Please see [this issue](https://github.com/hackseq/hackseq19/issues/2) and fill out [this doodle poll](https://doodle.com/poll/x3m5rau6yg9iz9x5) if you are still interested in organizing.

Let me know if you have any questions

**Comment by emreerhan** [comment]
Hi @SaberHQ @oganm @kfgibson @nedo0shki @jenjaelin ,

Check out the minutes for the first meeting [here](https://github.com/hackseq/hackseq19/issues/2) and if you're still interested in organizing, fill out [this doodle poll](https://doodle.com/poll/huhz6pxswgukmepq) for the second meeting. Make sure you're 'watching' the repo to keep up to date with new conversations.

Let me know if you have questions.

**Comment by bencardoen** [comment]
Hi @emreerhan , I've filled in the doodle, if there's something left to do in the organizational committee, count me in

**Comment by scatcher125** [comment]
@emreerhan did you send it again? I didn't get an email from github. Not sure what's wrong..
